src/repley/shadow/remote.cljs

```diff
@@ -0,0 +1,27 @@
+(ns repley.shadow.remote)
+
+(defn make-send [fetch]
+  (fn submit
+    ([value] (submit nil value))
+    ([{:keys [port mode]
+       :or   {port 3001 ;; TODO: use repley.config/default-config
```

Comment:
`repley.config` would need to be `.cljc`.

Reply:
Config is only data and a simple merging function, changed that to .cljc